diablo/models/sis_section.py

```diff
@@ -785,11 +782,12 @@ def _decorate_course_meeting_type(course):
 
 def _decorate_course_opt_in(course):
     # All non-APRX instructors must agree to opt in. If there are no non-APRX instructors, an admin opt-in suffices.
-    instructors_not_aprx = [i for i in course['instructors'] if i['roleCode'] != 'APRX']
+    all_instructors = course['instructors']
+    instructors_not_aprx = [i for i in all_instructors if i['roleCode'] != 'APRX']
     instructors_not_opted_in = [i for i in instructors_not_aprx if not i['hasOptedIn']]
     admin_opt_in = next((o for o in course['optIns'] if o['instructorUid'] == 'admin'), None)
 
-    if (len(instructors_not_aprx) and not len(instructors_not_opted_in)) or (not len(instructors_not_aprx) and admin_opt_in):
+    if (len(all_instructors) and not len(instructors_not_opted_in)) or admin_opt_in:
```

Comment:
@johncrossman I think the issue with this conditional is that an admin opt-in should qualify a course for scheduling only if there are one or more non-APRX instructors assigned to the course. As soon as a course gets a non-APRX instructor, the admin opt-in ceases to have any effect and the instructor opt-ins or lack thereof will control.

Reply:
Thanks @pauline2k.  Amendment coming soon.